frontend/src/routes/trade/Review.svelte

```diff
@@ -0,0 +1,130 @@
+        await goto(stepUrl(5, initialState));
+    }
+</script>
+
+<div class="block">
+    <div class="columns">
+        <div class="column" />
+        <div class="column is-two-thirds">
+            <h2 class="title is-2">Order Summary</h2>
+            <br />
+            <div class="box has-background-grey-dark">
+                <nav class="level">
+                    <div class="level-item has-text-centered">
+                        <div>
+                            <p class="heading">Side</p>
+                            <p class="title">{side}</p>
+                        </div>
+                    </div>
+                    <div class="level-item has-text-centered">
+                        <div>
+                            <p class="heading">Amount</p>
+                            <p class="title">{amount}</p>
+                        </div>
+                    </div>
+                    <div class="level-item has-text-centered">
+                        <div>
+                            <p class="heading">Type</p>
+                            <p class="title">
+                                {type === "PRICE" ? "$" : "UNITS"}
+                            </p>
+                        </div>
+                    </div>
+                    <div class="level-item has-text-centered">
+                        <div>
+                            <p class="heading">Ticker</p>
+                            <p class="title">{ticker}</p>
+                        </div>
+                    </div>
+                </nav>
+            </div>
+            <br />
+            <h3 class="title is-4">Articles</h3>
+            <div class="block">
+                <div class="tile is-ancestor">
+                    {#await articlesRequest}
+                        <div
+                            style="width: 100%; height: 5vh; display: flex; justify-content: center; align-items: center;">
+                            <p>Loading...</p>
+                        </div>
+                    {:then responses}
+                        <div class="tile is-parent is-vertical">
+                            {#each responses as response}
+                                <NewsTile
+                                    toggleArticle={null}
+                                    data={response}
+                                    description={false} />
+                            {/each}
+                        </div>
+                    {:catch error}
+                        <div
+                            style="width: 100%; height: 5vh; display: flex; justify-content: center; align-items: center;">
+                            <p>{error.message}</p>
+                        </div>
+                    {/await}
+                </div>
+            </div>
+            <br />
+            <h3 class="title is-4">Justification</h3>
+            <div class="box has-background-grey-dark content">
+                <p>{textEvidence}</p>
```

Comment:
Potentially, have _No textual justification has been provided._ as placeholder when textEvidence is empty - just to improve UI presentation.

Reply:
Makes sense - will do this for articles as well.